chain-connect/src/customClients/BrowserConnectClient.ts

```diff
@@ -12,7 +12,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { serialize } from "@gala-chain/api";
+import { SignatureDto, SigningScheme, serialize, signatures } from "@gala-chain/api";
```

Comment:
I'm not sure we want to add multisig feature to connect library yet -- too many things at once